crates/worker/src/partition/mod.rs

```diff
@@ -280,9 +284,10 @@ where
                 match res.as_ref() {
                     // run_inner never returns normally
                     Ok(_) => warn!("Shutting partition processor down because it stopped unexpectedly."),
-                    Err(ProcessorError::TrimGapEncountered { gap_to_lsn }) =>
+                    Err(ProcessorError::TrimGapEncountered { gap_to_lsn, sequence_number }) =>
                         info!(
                             trim_gap_to_lsn = ?gap_to_lsn,
+                            ?sequence_number,
```

Comment:
Can we replace `sequencer_number` with `read_pointer`? or if it's easier to understand "trim_gap_start", "trim_gap_end"?

Reply:
I like it, tyvm!